primitives/file-key-verifier/src/types.rs

```diff
@@ -15,6 +15,27 @@ pub struct FileKeyProof<const H_LENGTH: usize, const CHUNK_SIZE: u64, const SIZE
     pub proof: CompactProof,
 }
 
+/// Implement the `From<CompactProof>` trait for the `FileKeyProof` struct.
+impl<const H_LENGTH: usize, const CHUNK_SIZE: u64, const SIZE_TO_CHALLENGES: u64> From<CompactProof>
+    for FileKeyProof<H_LENGTH, CHUNK_SIZE, SIZE_TO_CHALLENGES>
+{
+    fn from(proof: CompactProof) -> Self {
+        Self {
+            file_metadata: Default::default(),
```

Comment:
I'm not sure I understand why you would convert a `CompactProof` into a `FileKeyProof` with default `file_metadata`

Reply:
I'm not a fan of this either, but it is a consequence of benchmarks being also tests. In tests, we use the mocked runtime, but in benchmarks we use the actual runtime. And I had to do this to get something to compile in benchmarks.